.gitignore

```diff
@@ -453,3 +453,4 @@ $RECYCLE.BIN/
 !.vscode/tasks.json
 !.vscode/launch.json
 !.vscode/extensions.json
+/src/Host/Configurations/mail.Development.json
```

Comment:
Could you please remove this line? It might affect other devs too.

Reply:
@xlogex  could you check this?

Reply:
@iammukeshm please see cumulative PR #321